src/css/custom.css

```diff
@@ -90,6 +90,15 @@
   display: none;
 }
 
+pre.shiki.nord > .code-container {
+  padding: var(--ifm-pre-padding);
+}
+
+pre.shiki.nord > .code-container > code {
+  display: block;
+  position: relative;
+}
```

Comment:
the error block is `position: absolute`, and the closest parent with `positon: relative` was too far and should've been the `code` tag itself.